frappe/tests/test_naming.py

```diff
@@ -10,12 +10,18 @@
 from frappe.model.naming import determine_consecutive_week_number, parse_naming_series
 
 class TestNaming(unittest.TestCase):
+	def setUp(self):
+		frappe.db.sql('delete from `tabNote`')
+		frappe.db.sql('delete from `tabToDo`')
+
 	def tearDown(self):
 		# Reset ToDo autoname to hash
 		todo_doctype = frappe.get_doc('DocType', 'ToDo')
 		todo_doctype.autoname = 'hash'
 		todo_doctype.save()
 
+		frappe.db.rollback()
```

Comment:
The deletes in SetUp, and resetting the ToDo.autoname (^) are inconsequential when you add the rollback. Maybe we can get rid of them?

Reply:
Retaining deletion for Note since it needs a unique title and other tests might have already created notes with the same title, which have to be deleted before running any test.